ait/core/server/broker.py

```diff
@@ -81,6 +87,52 @@ def _subscribe_all(self):
                 else:
                     self._subscribe(subscriber, plugin.name)
 
+        ## Lastly setup the outputstream to receive commands
+        self._subscribe_cmdr()
+
+    def _subscribe_cmdr(self):
+        """
+        Setup for the appropriate outbound stream that is configured to
+        accept command messages.  If no stream is specified, it looks
+        for the first outbound stream.
+        """
+
+        ## Only continue if command topic was supplied
+        if not self.command_topic:
+            return
+
+        cmd_sub_flag_field = 'cmd_subscriber'
+        cmd_stream = None
+
+        ##Lookup for outbound stream with subscribe flag set
+        cmd_streams = list((x for x in self.outbound_streams
+                       if hasattr(x, cmd_sub_flag_field) and
+                       getattr(x, cmd_sub_flag_field)))
+
+        cmd_stream = next(iter(cmd_streams), None)
+
+        ## Warn about multiple matches
+        if cmd_stream and len(cmd_streams) > 1:
+            log.warn('Multiple output streams found with {} field enabled, '
+                     '{} was selected as the default.'.format(
+                     cmd_sub_flag_field, cmd_stream.name))
+
+
+        ## Do we want to make a guess for the command stream?
+        ## Just grab the first output stream
+        if not cmd_stream and self.command_stream_select_defaut:
```

Comment:
Per the above, I say we drop the `and self.command_stream_select_default` from the check